documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -864,7 +864,7 @@ EOF
 ====
 * If the migrated machines are set with a custom CPU model, they will be set with that CPU model in the destination cluster, regardless of the setting of `preserveClusterCpuModel`.
```

Comment:
```suggestion
* If the migrated machines is set with a custom CPU model, it will be set with that CPU model in the destination cluster, regardless of the setting of `preserveClusterCpuModel`.
```